app/controllers/api/v1/courses/enrollments_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Api
+  module V1
+    module Courses
+      class EnrollmentsController < ApplicationController
+        expose :course
+        expose :enrollment, -> { Enrollment.find_by(enrollment_params) }
```

Comment:
Why can't we find enrollment by id?

Reply:
We can't because there is no id passed as a path variable